app/models/middleware_performance.rb

```diff
@@ -19,7 +19,12 @@ def self.build_results_for_report_middleware(options)
     results = []
 
     model_class.find_each do |ms|
-      raw_stats = fetch_raw_stats(ms, start_time, end_time, interval)
+      begin
+        raw_stats = fetch_raw_stats(ms, start_time, end_time, interval)
+      rescue ::Hawkular::BaseClient::HawkularConnectionException => exception
+        $log.log_backtrace(exception)
```

Comment:
why is this backtrace and not $log.error? (not necessarily wrong, just wondering if there's specific reason that an exception is not logged in error)